Makefile.common

```diff
@@ -14,6 +14,8 @@
 # Executables
 ###########################################################################
 
+COQBIN:=$(CURDIR)/bin/
```

Comment:
I prefer `?=`. For example:

```
gares@gargamel:~$ cd /tmp/
gares@gargamel:/tmp$ cat > Makefile
X:=3
all:
	echo $(X)
gares@gargamel:/tmp$ make
echo 3
3
gares@gargamel:/tmp$ make X=4
echo 4
4
gares@gargamel:/tmp$ X=4 make
echo 3
3
gares@gargamel:/tmp$ cat > Makefile
X?=3
all:
	echo $(X)
gares@gargamel:/tmp$ make
echo 3
3
gares@gargamel:/tmp$ make X=4
echo 4
4
gares@gargamel:/tmp$ X=4 make
echo 4
4
gares@gargamel:/tmp$ 
```

Reply:
I prefer `:=` so that I don't have to worry about unclean environment.